src/nix-store/nix-store.cc

```diff
@@ -940,11 +940,15 @@ static void opServe(Strings opFlags, Strings opArgs)
                 auto deriver = readString(in);
                 if (deriver != "")
                     info.deriver = store->parseStorePath(deriver);
-                info.narHash = Hash(readString(in), htSHA256);
+                info.narHash = Hash(readString(in), HashType::SHA256);
                 info.references = readStorePaths<StorePathSet>(*store, in);
                 in >> info.registrationTime >> info.narSize >> info.ultimate;
                 info.sigs = readStrings<StringSet>(in);
-                in >> info.ca;
+                {
+                    std::string rawCA;
+                    in >> rawCA;
+                    info.ca = parseContentAddressOpt(rawCA);
```

Comment:
```suggestion
                    info.ca = parseContentAddressOpt(readString(in));
```

Reply:
GitHub has multi-line suggestions now, can you use those @matthewbauer?